services/src/performance-tests/src/main/groovy/org/openkilda/performancetests/helpers/TopologyHelper.groovy

```diff
@@ -50,6 +50,15 @@ class TopologyHelper extends org.openkilda.functionaltests.helpers.TopologyHelpe
             def dst = topo.pickRandomSwitch([src])
             topo.addIsl(src, dst)
         }
+
+        def allSwitches = topo.getSwitches()
+        def allLinks = topo.getIsls()
+        for (def i = 0; i < allSwitches.size() - 1; i++) {
+            if (!allLinks.find { it.srcSwitch == allSwitches[i] && it.dstSwitch == allSwitches[i+1] }) {
+                topo.addIsl(allSwitches[i], allSwitches[i+1])
+            }
```

Comment:
else?

Reply:
Links are already added a little bit above.
Here we just create link between neighbor switches.
'else' in this place is needless